### PR TITLE
[EICNET-580]feature: Add facets languages and roles + sorts fort last active user

### PR DESCRIPTION
**Pull request by Dromadoros**
How to test :

- [x] Go to /people
- [x] Check if there are language  + role filter and last active sort option

### CODE REVIEW

File: config/sync/search_api.index.global.yml [hunk 1]
```diff
@@ -743,9 +752,6 @@ processor_settings:
     weights:
       preprocess_query: -30
   entity_status: {  }
-  flag_count_indexer:
-    flag_index:
-      - recommend_group
```
Comment (4hardfire) [inline config/sync/search_api.index.global.yml:748]: @Dromadoros Is this ok or was it a mistake?

**Review by 4hardfire — COMMENTED** [review]
Check my last comment before I approve it.

**Review by 4hardfire — CHANGES_REQUESTED** [review]
- [ ] Revert config

File: config/sync/search_api.index.global.yml (hunk 1 above)
Comment (Dromadoros) [reply config/sync/search_api.index.global.yml]: Done

**Review by 4hardfire — APPROVED** [review]
Seems ok to me!

**Pull request merged**